wagtail/admin/wagtail_hooks.py

```diff
@@ -874,6 +874,15 @@ def is_shown(self, request):
         )
 
 
+class PageTypesReportMenuItem(MenuItem):
+    def is_shown(self, request):
+        return getattr(
+            settings, "WAGTAIL_PAGE_TYPES_ENABLED", True
```

Comment:
I'm not sure I recognise this setting. Is it new?

Reply:
Nope, it's been there since early commits (https://github.com/wagtail/wagtail/pull/10850/commits/c81157633206b46977784b90a63499e2dcf8d5a4). It's changed now that I've added the permission check as well

Reply:
I suggest this should be renamed to `WAGTAIL_PAGE_TYPES_REPORT_ENABLED` or similar. I guess you followed the pattern of [`WAGTAIL_WORKFLOW_ENABLED`](https://docs.wagtail.org/en/stable/reference/settings.html#wagtail-workflow-enabled) from the workflow report here, but that setting is for enabling/disabling workflows as an overall concept, not just the one report.

Reply:
I followed the pattern from the aging pages report:

```python
class AgingPagesReportMenuItem(MenuItem):
    def is_shown(self, request):
        return getattr(
            settings, "WAGTAIL_AGING_PAGES_ENABLED", True
        ) and PagePermissionPolicy().user_has_any_permission(
            request.user, ["add", "change", "publish"]
        )
```

Sure, I can rename it

Reply:
I've renamed it to `WAGTAIL_PAGE_TYPES_USAGE_REPORT_ENABLED` (a mouthful)

Reply:
Yeah, I think the name stumped me a little too - `WAGTAIL_PAGE_TYPES_ENABLED` sounds a lot like something that would impact functionality more widely. The new name sounds more like what it does :D